packages/shared/src/components/onboarding/OnboardingFeedHeader.tsx

```diff
@@ -62,12 +65,16 @@ export const OnboardingFeedHeader = ({
       event_name: AnalyticsEvent.CreateFeed,
     });
 
-    return router.replace({
-      pathname: router.route,
-      query: {
-        welcome: 'true',
-      },
-    });
+    return router.replace(
+      isExtension
+        ? `${window.location.href}?welcome=true`
+        : {
+            pathname: router.route,
+            query: {
+              welcome: 'true',
+            },
+          },
+    );
```

Comment:
Alternatively we could update `CustomRouter` to accept objects and deconstruct them when a object is passed.
https://github.com/dailydotdev/apps/blob/50192c8ce187519e01144e506a17cfbcce8f91af/packages/extension/src/lib/CustomRouter.ts#L56-L59

Reply:
Will re-do it and use the same util that next.js uses (and exports)
https://github.com/vercel/next.js/blob/473ef817a77c1103e034cb3dcb1c8b10c1bc9e9d/packages/next/src/shared/lib/router/utils/format-url.ts#L29-L73

Reply:
Just checking, there is another place where we set welcome=true, would that need the same fix for the extension? If yes, does it make sense to do it in this PR?

Reply:
The other place is on the onboarding page, which does not exist in the extension.
But if I implement formatUrl from next into our CustomRouter, it wouldn't be an issue anyways

Reply:
I implemented it, but it didn't really have any good effect. It worked fine as intended, it set the `?welcome=true` when I asked it to. But it doesn't persist in `router.query`.

Had a [chat with Sab](https://dailydotdev.slack.com/archives/C068F77M8Q7/p1709639575360589?thread_ts=1709636486.310309&cid=C068F77M8Q7), and the easiest way is just to send them to webapp and not spend too much time, as we might remove this welcome banner after a future experiment anyways.
And it's a edge case flow too.